src/org.ts

```diff
@@ -27,6 +29,40 @@ function validateOrgData(
   return undefined;
 }
 
+async function removeLastRelatedMember(
+  orgName: string,
+  options: ServiceOptions,
+): Promise<Result<undefined, OrgProblemType>> {
+  const memberModel = getModelForClass(MemberModelSchema, {
+    existingConnection: options.dbConnection,
+  });
+
+  const memberCount = await memberModel.count({ orgName });
+  if (memberCount > 1) {
+    options.logger.info({ orgName }, 'Refused deletion - existing org members');
+    return {
+      didSucceed: false,
+      reason: OrgProblemType.EXISTING_MEMBERS,
+    };
+  }
+
+  if (memberCount === 1) {
+    const lastAdmin = await memberModel.findOne({ orgName, role: Role.ORG_ADMIN });
+    if (lastAdmin === null) {
+      options.logger.info({ orgName }, 'Refused deletion - last member not org admin');
```

Comment:
Similar to the above:

```suggestion
      options.logger.info({ orgName }, 'Refused org deletion because last member is not admin');
```